pkg/models/expense_classification.go

```diff
@@ -16,3 +21,19 @@ type InvoicesExpensesClassificationDetails struct {
 	LineNumber                       uint                         `xml:"lineNumber"`
 	ExpensesClassificationDetailData []ExpensesClassificationType `xml:"ExpensesClassificationDetailData"`
 }
+
+func (classification ExpensesClassificationType) MarshalXML(enc *xml.Encoder, start xml.StartElement) error {
+	type expensesClassificationType struct {
+		ClassificationType     string  `xml:"ecls:classificationType"`
+		ClassificationCategory string  `xml:"ecls:classificationCategory"`
+		Amount                 float64 `xml:"ecls:amount"`
+		ID                     *byte   `xml:"ecls:id"`
+	}
+
+	err := enc.EncodeElement(expensesClassificationType(classification), start)
+	if err != nil {
+		return fmt.Errorf("xml marshal expense classification: %w", err)
+	}
+
+	return nil
+}
```

Comment:
Notice i made a new duplicate struct of the `ExpenseClassificationType` here with just the namespace prefix added `ecls:`.
The reason for this is because I came across an issue when marshaling/unmarshaling `ExpensesClassificationType`  &. `IncomeClassificationType` xml with namespaces and then trying to retrieve them with `RequestDocs`
https://github.com/golang/go/issues/11496
https://github.com/golang/go/issues/9519